testgrid/config.yaml

```diff
@@ -3847,78 +3847,113 @@ dashboards:
       alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
     alert_stale_results_hours: 24
     num_failures_to_alert: 6 # Runs every 2h. Alert when it's been failing for 12 hours
+# kubeadm x-on-y tests
```

Comment:
lines from 3850 to 3879 are not new, but only a weird diff visualization :-(
In the previous version, the same lines are 3892-3900 and 3905-3021